change/@ni-nimble-blazor-eed57bcb-a0ee-4959-bdc1-c9e233274889.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Createt Blazor Wrapper for Wafer Map Component",
+  "packageName": "@ni/nimble-blazor",
+  "email": "33986780+munteannatan@users.noreply.github.com",
```

Comment:
@msmithNI as @jattasNI and @atmgrifter00 are OOO can you give this PR a final review as Blazor code owner?

In particular I'm interested to know if the `GetValidityAsync` is a reasonable approach or if you are aware of other patterns we use. i.e. it is clicking a button via playwright, exercising the api under test in blazor and writing to a text field, and reading the result in the text field via playwright.

Reply:
The structure of that test looks reasonable to me, it's pretty similar to what we have for the others like DrawerTests/ DialogTests.

I'd expect most of the Blazor acceptance tests to look pretty similar:
- Load page with component, do any initial asserts (assert component is visible, for example)
- Interact with the component via Playwright actions. Either direct interaction, or via other buttons that trigger component APIs (like what this one is doing)
- Assert based on new component state, and/or assert that other controls got updated based on component logic that executed (like this one)

For this test, my only concern would be that it might pass even if the wafer map hasn't rendered, from what I can tell.